Solution1/WeatherApi/Configuration/Config.cs

```diff
@@ -0,0 +1,17 @@
+﻿using BusinessLayer.Configuration.Abstract;
+
+namespace WeatherApi.Configuration
+{
+    public class Config : IConfig
```

Comment:
don't inherit from interface, again you not need it, and clear unnecessary fields from IConfig